guides/developer_guide.md

```diff
@@ -78,8 +78,8 @@ npm run docker
 ```
 
 ### Backend
-The KFP backend with Tekton uses a modified version of Kubeflow Pipelines api-server, persistent agent, and metadata writer. 
-1. To build these two images, clone this repository under the [GOPATH](https://golang.org/doc/gopath_code.html#GOPATH) and rename it to `pipelines`. 
+The KFP backend with Tekton uses a modified version of Kubeflow Pipelines api-server, persistent agent, and metadata writer.
+1. To build these two images, clone this repository under the [GOPATH](https://golang.org/doc/gopath_code.html#GOPATH) and rename it to `pipelines`.
```

Comment:
From my testing, seems we don't need to rename it to `pipelines`, could you double confirm? Thanks

Reply:
For building the docker image, the repo name doesn't matter. For building the go binaries, it depends on the user's GOPATH. Some users may see errors if their gopath is not matching with the directory/package path.